weave-js/src/components/Panel2/ChildPanelExportReport/utils.ts

```diff
@@ -82,3 +83,27 @@ export function formatUpdatedAt({
   // Handles when unit is "week", "month"
   return format(date, 'MMM dd');
 }
+
+/**
+ * Generates an empty config for a new slate report
+ * @param initialBlocks (optional) - initial contents for the report body
+ */
+export function getEmptyReportConfig(
+  initialBlocks: WeavePanelSlateNode[] = []
+) {
+  return {
+    blocks: [...initialBlocks, {type: 'paragraph', children: [{text: ''}]}],
+    discussionThreads: [],
+    panelSettings: {
```

Comment:
I'm not very familiar with reports. It seems a little odd to me that this plot specific stuff is here since the panel export seemed more general.

Reply:
I believe these default panel settings are necessary because the user may later add workspace panels to a report created here. At some point we should probably refactor some of this report creation logic to the backend so it's not duplicated across all our clients (core, weave, sdk)